crates/holochain/tests/tests/regression/mod.rs

```diff
@@ -2,6 +2,39 @@ use holo_hash::ActionHash;
 use holochain::sweettest::{SweetAgents, SweetConductor, SweetDnaFile};
 use holochain_wasm_test_utils::TestWasm;
 
+// make sure the wasm cache at least creates files
+#[tokio::test(flavor = "multi_thread")]
+async fn wasm_disk_cache() {
+    holochain_trace::test_run();
+    let mut conductor = SweetConductor::from_standard_config().await;
+
+    let (dna_file, _, _) =
+        SweetDnaFile::unique_from_test_wasms(vec![TestWasm::ValidateRejectAppTypes, TestWasm::Crd])
+            .await;
+    let agent = SweetAgents::alice();
+
+    let (cell,) = conductor
+        .setup_app_for_agent("app", agent, &[dna_file])
+        .await
+        .unwrap()
+        .into_tuple();
+
+    let _created: ActionHash = conductor
+        .call(
+            &cell.zome(TestWasm::Crd.coordinator_zome_name()),
+            "create",
+            (),
+        )
+        .await;
+
+    let mut cache_dir = conductor.db_path().to_owned();
+    cache_dir.push("wasm-cache");
```

Comment:
Public const above and use here?

Reply:
done: https://github.com/holochain/holochain/pull/4004/commits/c94447cd5781be7212595a64ac06d9da1e37acc5